.github/workflows/linux.yml

```diff
@@ -33,18 +33,21 @@ jobs:
           mkdir builds
           cd builds
       - name: Installing packages
-        run: |      
+        run: |
           python3.8-dbg -c 'import sys; print("Python debug build:", hasattr(sys, "gettotalrefcount"))'
           python3.8-dbg -m pip install --upgrade pip setuptools wheel
           python3.8-dbg -m pip install --upgrade numpy cython pytest pytest-xdist pybind11
           python3.8-dbg -m pip install --upgrade mpmath gmpy2==2.1.0rc1 pythran threadpoolctl
           python3.8-dbg -m pip uninstall -y nose
           cd ..
       - name: Building SciPy
-        run: python3.8-dbg -u runtests.py -g -j2 --build-only
+        run: |
+          export SETUPTOOLS_USE_DISTUTILS=stdlib
```

Comment:
It's either this, or just permanently pin to `<60.0`. I'm fine with trying this, if there are more problems we should move to the permanent pin. We plan to drop `setuptools` completely, so there's not much point in continuing to patch it up.